app/modules/users/models.py

```diff
@@ -238,6 +238,95 @@ def __repr__(self):
             ')>'.format(class_name=self.__class__.__name__, self=self)
         )
 
+    @classmethod
```

Comment:
Is this intentional or a merge error? I removed this on the request of @bluemellophone